EventListener/PjaxBrowserLanguageSubscriber.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformUIBundle\EventListener;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * On PJAX requests, sets the request's locale from the browser's accept-language header.
+ */
+class PjaxBrowserLanguageSubscriber implements EventSubscriberInterface
```

Comment:
In general I would remove the Pjax to something more generic like LocalSubscriber so it would work even if it's not pjax.

And if in the future we have to has a condition based on user settings then it's not only brower languages